PWGCF/FemtoUniverse/Tasks/femtoUniversePairTaskTrackTrackExtended.cxx

```diff
@@ -267,17 +304,33 @@ struct femtoUniversePairTaskTrackTrackExtended {
         case 310:  // Kaon 0 SHORT
           return IsKaonNSigma(mom, nsigmaTPCK, nsigmaTOFK);
           break;
+        case 1000010020:  // Deuteron+
+        case -1000010020: // Deuteron-
+          if (lincut.ConfIsLine == true) {
```

Comment:
In the case for deuterons please leave only return IsDeuteronNSigma (move this to the IsDeuteronNSigma function)